tests/contract_testcases/solana/issue731-2.sol

```diff
@@ -0,0 +1,23 @@
+pragma solidity 0.6.12;
```

Comment:
How about naming the file as `casting_desctructure.sol`. It is not easy to look at your file and guess what you're testing. I think you can mention the issue in the file itself.

Reply:
I've given them both the best names I could come with and they're still not very descriptive. The name `issue731.sol` is very clear: have a look at issue 731.

Also I don't think the file name itself is also not terribly important. Only if the test fails do you need to figure out what it is testing; and even then, the test could fail for any reason if there is a code change.